Sources/SpeziOnboarding/OnboardingConsentView.swift

```diff
@@ -30,6 +33,7 @@ import SwiftUI
 ///         // The action that should be performed once the user has provided their consent.
 ///     },
 ///     title: "Consent",   // Configure the title of the consent view
+///     identifier: "MyFirstConsentForm", // Specify a unique identifier for the consent form, helpful for distinguishing consent forms when storing.
```

Comment:
As noted, it would be great to use the enum approach in the docs to highlight this. Maybe we can even highlight this as a `tip` in the DocC documentation as a best practice?

Reply:
I added a tip/explanation to the comments.